client/queue/Queue.tsx

```diff
@@ -76,6 +80,24 @@ export class Queue extends React.Component<Props, {}> {
             }
           </dd>
 
+          <dt>Max Dispatches Per Second</dt>
```

Comment:
I think it's ok to hide these labels when not configured.

Reply:
Improved as your suggestion https://github.com/fireworq/fireworqonsole/pull/400/commits/990a210921cde423447b9e94fd4ad38e3661234f